index.html

```diff
@@ -5,41 +5,57 @@
   <meta charset="utf-8">
 
   <title>Sprint Challenge - Home</title>
-
+  <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.2.0/css/all.css" integrity="sha384-hWVjflwFxL6sNzntih27bfxkr27PmbbK/iSvJ+a4+0owXq79v+lsFkW54bOGbiDQ" crossorigin="anonymous">
   <link href="https://fonts.googleapis.com/css?family=Roboto|Rubik" rel="stylesheet">
   <link rel="stylesheet" href="css/index.css">
 
 </head>
 
 <body>
     <div class="container">
+        <nav>
+            <img src = "img/lambda-black.png">
+            <a href = "#">Home</a>
+            <a href = "about.html">About</a>            
+            <a href = "#">Prodcucts</a>
+            <a href = "#">Blog</a>
+            <a href = "contact.html">Contact</a>
+            <i class="fas fa-bars"></i>
+        </nav>
+
+        <img class ="jumboImg" src = "img/jumbo.jpg">
+    
+        </nav>
    
         <section class="top-content">
+
             <div class="text-container">
                 <h2>The Future</h2>
                 <p>Proin sed quam sed tellus vestibulum ultrices quis in nunc. Phasellus id dui id tortor tincidunt efficitur. Proin faucibus imperdiet erat, non varius lacus. Maecenas non nisl id turpis egestas tincidunt. Nam condimentum venenatis magna eget finibus.</p>
             </div>
+
             <div class="text-container">
                 <h2>The Past</h2>
                 <p>Proin sed quam sed tellus vestibulum ultrices quis in nunc. Phasellus id dui id tortor tincidunt efficitur. Proin faucibus imperdiet erat, non varius lacus. Maecenas non nisl id turpis egestas tincidunt. Nam condimentum venenatis magna eget finibus.</p>
             </div>
+            
         </section>
         
         <section class="middle-content">
         
             <h2>Why Did It Have To Be Boxes...</h2>
             
             <div class="boxes">
-                <div class="box">Box 1</div>
-                <div class="box">Box 2</div>
-                <div class="box">Box 3</div>
-                <div class="box">Box 4</div>
-                <div class="box">Box 5</div>
-                <div class="box">Box 6</div>
-                <div class="box">Box 7</div>
-                <div class="box">Box 8</div>
-                <div class="box">Box 9</div>
-                <div class="box">Box 10</div>
+                <div id = "teal" class="box">Box 1</div>
+                <div id = "gold"class="box">Box 2</div>
+                <div id = "cadetblue" class="box">Box 3</div>
+                <div id = "coral" class="box">Box 4</div>
+                <div id = "crimson" class="box">Box 5</div>
+                <div id = "forestgreen" class="box">Box 6</div>
+                <div id = "darkorchid" class="box">Box 7</div>
+                <div id = "hotpink" class="box 8">Box 8</div>
+                <div id = "indigo" class="box 9">Box 9</div>
+                <div id = "dodgerblue" class="box 10">Box 10</div>
```

Comment:
So this works, but I would use the id's sparingly. The was to do this in convention is to add another class. The reason being is because all of the boxes look like the class `box` and then you are only changing the color of the box. So the best way to do that is to add another class with a new color.

For Example

```<div id = "teal" class="box">Box 1</div>```
Should be
```<div class="box box1">Box 1</div>```
And 
```<div id = "gold"class="box">Box 2</div>```
Should be 
```<div class="box box2">Box 2</div>```

Notice how I just made another class instead of an id. Now in your css you can just style it like normal. 

Your way absolutely work! It's just as best practices, use another class to override styles on one class. You'll see why once we get to JavaScript 😉